app/models/assistance_request.rb

```diff
@@ -13,6 +14,7 @@ class AssistanceRequest < ActiveRecord::Base
   }
   scope :oldest_requests_first, -> { order(:start_at) }
   scope :requested_by, -> (user) { where(:requestor => user) }
+  scope :code_reviews, -> {where(:type => 'CodeReviewRequest')}
```

Comment:
Don't need this line, should use the `CodeReviewRequest` model directly for querying